### PR TITLE
Additions and reordering best practices 

### PR DESCRIPTION
**Pull request by 0kok0**

### CODE REVIEW

File: drafts/personal-security-checklist.md [hunk 1]
```diff
@@ -6,20 +6,44 @@
...
+- [ ] Get a password manager (default: bitwarden)
     - [ ] secure with 2FA (YubiKey / Google Auth)
-- [ ] Remove unused browser extensions
-- [ ] Disable infrequently used browser extensions
+- [ ] Disable unused/infrequently used browser extensions
 - [ ] If your computer is old, start fresh
     - [ ] fresh OS install
     - [ ] new computer
+- [ ] Do not tamper with the integrity protection of your OS (OSX SIP, Linux App Armor)
+    - [ ] Bonus: Get a vm and use put your browser and mail client in the vm.
```
Comment (corpetty) [inline drafts/personal-security-checklist.md:25]: grammar issue here.

File: drafts/personal-security-checklist.md [hunk 2]
```diff
@@ -6,20 +6,44 @@
...
 - [ ] Review launch-on-startup software
+
+### Mobile
+- [ ] Get Trail of Bits iVerify.
+    - [ ] Go through the tutorials and change the settings accordingly.
+- [ ] Do not charge your phone with a charger that is not yours/from someone you trust.
```
Comment (corpetty) [inline drafts/personal-security-checklist.md:31]: `from someone you trust` seems to be wrong here.

File: drafts/personal-security-checklist.md [hunk 3]
```diff
@@ -6,20 +6,44 @@
...
+- [ ] Get a phone that supports yubikey plugable/nfc
+- [ ] Get a vpn for your phone. Nordvpn works.
```
Comment (corpetty) [inline drafts/personal-security-checklist.md:33]: why specify Nord?

Comment (0kok0) [reply drafts/personal-security-checklist.md:33]: Not a specification, just as recommandation if folks don't wanna go through the effort of researching their own thing but sure questionable

**Pull request merged**